src/SfxWeb/src/app/Models/eventstore/timelineGenerators.ts

```diff
@@ -238,19 +240,29 @@ export abstract class TimeLineGeneratorBase<T> {
                         config.relevantEventsType.forEach(relevantEventType => {
 
                             // check if event type is relevant
+                            if(iterEvent instanceof RepairTask) {
```

Comment:
can we move this information onto the repair task event that gets generated from the timeline generator for repair tasks.

Reply:
I don't think so. Thats why I was saying that we would need to add kind to the event that originally comes in. This is currently the best way to do it for the POC until we can add more information to events